vendors/microchip/boards/ecc608a_plus_winsim/aws_demos/config_files/atca_config.h

```diff
@@ -28,17 +28,30 @@
 #ifndef _ATCA_CONFIG_H
 #define _ATCA_CONFIG_H
 
-/** Use I2C */
+ /** Use HID */
 #define ATCA_HAL_KIT_HID
+#define ATCA_ATECC608A_SUPPORT
 
 /** Use the following address for ECC devices */
-#define ATCA_I2C_ECC_ADDRESS    0x6C
+#define ATCA_I2C_ECC_ADDRESS           0xC0
+#define ATCA_DEVICE_PRIVATE_KEY_SLOT   0x00
```

Comment:
Does it need update for EPS32 ECC608 config files ?
https://github.com/aws/amazon-freertos/blob/main/vendors/espressif/boards/esp32/aws_demos/config_files/ecc608a_pkcs11_config/atca_config.h

Reply:
I believe the address should stay the same for other platforms. Perhaps we should change this board name to ecc608b_plus_winsim.

Reply:
The macro like `ATCA_DEVICE_PRIVATE_KEY_SLOT ` needs to be defined in `https://github.com/aws/amazon-freertos/blob/main/vendors/espressif/boards/esp32/aws_demos/config_files/ecc608a_pkcs11_config/atca_config.h` or in a common header file. Its been used in `libraries/abstractions/pkcs11/ecc608a/atca_cert_chain.c` which I guess is common for both platforms.